lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
I think the operative bit there is "in a language that doesn't use pattern matching" :smile: 

With pattern matching, we can be a lot more efficient at capturing our intent. "Here's an ID; give me the user, please", rather than "I would like you to retrieve a user for me by this method, please". We can be declarative, and our designs can take advantage of that.